tests/Data/Formatter/XmlDataResponseFormatterTest.php

```diff
@@ -39,4 +39,111 @@ public function testFormatterEncoding(): void
         );
         $this->assertSame(['application/xml; ISO-8859-1'], $result->getHeader('Content-Type'));
     }
+
+    public function testFormatterVersion(): void
+    {
+        $factory = new Psr17Factory();
+        $dataResponse = new DataResponse('test', 200, '', $factory);
+        $formatter = new XmlDataResponseFormatter();
+        $formatter = $formatter->withVersion('1.1');
+        $result = $formatter->format($dataResponse);
+        $result->getBody()->rewind();
```

Comment:
Is it really needed?

Reply:
Yes. See https://github.com/php-fig/http-message/blob/master/src/StreamInterface.php#L137.